docs/translations/translations.json

```
@@ -85,6 +85,9 @@
   "toggleRTL": "Toggle right-to-left/left-to-right",
   "toggleTheme": "Toggle light/dark theme",
   "traffic": "Traffic",
+  "v5IsOut": "v5.beta is out! 🎉 Head to the",
+  "v5docsLink": "next documentation",
```

```suggestion
  "v5docsLink": "v5 documentation",
```

Let's just call it "v5 documentation" for now. `next` is only our internal name for the branch.